schema/generate.sh

```diff
@@ -1,37 +1,25 @@
 #!/bin/bash
 
-cd $(dirname "$0")
```

Comment:
Will this bash script work now if you run it from the `schema` folder? I think it will only work if you run it from the root folder due to the relative path to generated file.

Reply:
yes - works for me